cluster/config-defaults.yaml

```diff
@@ -44,6 +44,8 @@ kube_aws_ingress_default_lb_type: "application"
 #
 # "exec": same as "pre" and kube-ingress-aws-controller will default to NLB
 # after removing it you need to set kube_aws_ingress_default_lb_type: "network" above and cleanup skipper and ingress-ctl deployment.yaml
+# TODO(sszuecs): grace period need also to be set for NLB and in skipper deployment we should clean i up
+# skipper_termination_grace_period: 302
```

Comment:
Not sure I fully understand the comment. It sounds like that behind NLB termination grace period should be 302 seconds instead of default 70 we have now:
https://github.com/zalando-incubator/kubernetes-on-aws/blob/33a818d170340955fa30027af29b277a3e899a06/cluster/config-defaults.yaml#L102-L104

Should we rather increase default value to 302 unconditionally?

Reply:
we change it to 302s if the majority of the high traffic clusters are migrated